sdcm/cluster.py

```diff
@@ -196,6 +198,15 @@ def set_test_id(cls, test_id):
         else:
             LOGGER.warning("TestID already set!")
 
+    @classmethod
+    def tester_obj(cls):
```

Comment:
it is not used anywhere in this PR. Is there any reason we need it?

Reply:
We add for having global way to get to the tester object to add failures or error. (I was in use until this morning :))

Reply:
so please remove this